content/getting-started/yaml.md

```diff
@@ -401,3 +401,30 @@ workflows:
             excludes:
               - '**/*.md'
 ```
+
+## Working directory
+
+You may select working directory globally for the entire workflow or for locally individual scripts. If not specified, global working directory defaults to the directory where the repository is cloned (`/Users/builder/clone`). You can override global working directory by specifying working directory in the individual steps. Consider the example below:
+
+```yaml
+workflows:
+  build-ios:
+    name: Build iOS and Android
+    working_directory: mobile/
+    scripts:
+      - name: Prepare
+        script: # current working directory is /Users/builder/clone/mobile
+      - name: Build iOS
+        script: # iOS build command
+        working_directory: mobile/ios
```

Comment:
should all working directory paths end with a slash for consistency? compare: mobile/ and mobile/ios

Reply:
not necessarily, slash is optional

Reply:
I removed slash from `mobile/`